Opening Weekly Reports page with Admin raises NullPointerException

<!--
If you've never submitted an issue to the SORMAS repository before or this is your first time using this template, please read the Contributing guidelines (https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/docs/CONTRIBUTING.md) for an explanation of the information we need you to provide. You don't have to remove this comment or any other comment from this issue as they will automatically be hidden.
-->
### Bug Description
<!-- Mandatory -->
When opening Weekly Reports with user that has only Admin as user role then a Server error is shown (NullPointerExpection)

### Steps to Reproduce
<!-- Optional; please add more steps if necessary -->
1. Login with user with role Admin only.
2. Click on Reports in the menu.

### Actual Behavior
Server Error is shown (NPE)

### Expected Behavior
<!-- Optional -->
Admin has WEEKLYREPORT_VIEW right [(SORMAS_User_Rights.xlsx)](https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/sormas-api/src/main/resources/doc/SORMAS_User_Rights.xlsx) and should be able open the Weekly Report page without errors.

### Screenshots
<!-- Optional -->
![WeeklyReports-Admin-NPE](https://user-images.githubusercontent.com/95228108/166883850-3479f957-9cab-45c1-bd56-4b9dd5f45e8e.PNG)

### System Details
<!-- Mandatory; you only have to specify the Server URL if the error appeared on a publicly available test server -->
* Device: 
* SORMAS version: 
  * Local: Version: 1.71.0-SNAPSHOT (b900a16)
  * test.sormas.netzlink.com: Version: 1.71.0-SNAPSHOT (ac57983)
* Android version/Browser: Chrome 
* Server URL:
  * Local
  * test.sormas.netzlink.com
* User Role: Admin

### Additional Information
<!-- Optional -->
ServerLogs (local):
[WeeklyReports-Admin-NPE.log](https://github.com/hzi-braunschweig/SORMAS-Project/files/8629549/WeeklyReports-Admin-NPE.log)
```
	at de.symeda.sormas.api.report.__WeeklyReportFacade_Remote_DynamicStub.getSummariesPerOfficer(de/symeda/sormas/api/report/__WeeklyReportFacade_Remote_DynamicStub.java)
	at de.symeda.sormas.api.report._WeeklyReportFacade_Wrapper.getSummariesPerOfficer(de/symeda/sormas/api/report/_WeeklyReportFacade_Wrapper.java)
	at de.symeda.sormas.ui.reports.WeeklyReportOfficersGrid.reload(WeeklyReportOfficersGrid.java:173)
	at de.symeda.sormas.ui.reports.ReportsView.reloadGrid(ReportsView.java:160)
	at de.symeda.sormas.ui.reports.ReportsView.enter(ReportsView.java:152)
...
Caused by: java.lang.NullPointerException
	at de.symeda.sormas.backend.report.WeeklyReportFacadeEjb.getSummariesPerOfficer(WeeklyReportFacadeEjb.java:217)
```

`WeeklyReportFacadeEjb.getSummariesPerOfficer` runs into an NPE because of missing `Region`. It seems that `ReportsView.java:63` decides that the user is not on national level and therefore the following backend logic expects a region that the user apperently does not have. I guess the user also another jurisdiction level by roles which results in a lower jurisdiction level role to win `UserRole.getJurisdictionLevel.java:426`

--- a/sormas-backend/src/main/java/de/symeda/sormas/backend/report/WeeklyReportFacadeEjb.java
+++ b/sormas-backend/src/main/java/de/symeda/sormas/backend/report/WeeklyReportFacadeEjb.java
@@ -157,7 +157,7 @@ public class WeeklyReportFacadeEjb implements WeeklyReportFacade {
 	public List<WeeklyReportRegionSummaryDto> getSummariesPerRegion(EpiWeek epiWeek) {
 		JurisdictionLevel jurisdictionLevel = userService.getCurrentUser().getJurisdictionLevel();
 
-		if (!isNationOrNoneJurisdiction(jurisdictionLevel)) {
+		if (jurisdictionLevel != JurisdictionLevel.NONE & jurisdictionLevel != JurisdictionLevel.NATION) {
 			return new ArrayList<>();
 		}
 
@@ -204,10 +204,6 @@ public class WeeklyReportFacadeEjb implements WeeklyReportFacade {
 		}
 
 		return summaryDtos;
-	}
-
-	public boolean isNationOrNoneJurisdiction(JurisdictionLevel jurisdictionLevel) {
-		return jurisdictionLevel == JurisdictionLevel.NONE || jurisdictionLevel == JurisdictionLevel.NATION;
 	}
 
 	@Override

--- a/sormas-backend/src/main/java/de/symeda/sormas/backend/report/WeeklyReportFacadeEjb.java
+++ b/sormas-backend/src/main/java/de/symeda/sormas/backend/report/WeeklyReportFacadeEjb.java
@@ -207,8 +207,8 @@ public class WeeklyReportFacadeEjb implements WeeklyReportFacade {
 	}
 
 	public boolean isNationOrNoneJurisdiction(JurisdictionLevel jurisdictionLevel) {
-       return jurisdictionLevel == JurisdictionLevel.NONE || jurisdictionLevel == JurisdictionLevel.NATION;
-    }
+		return jurisdictionLevel == JurisdictionLevel.NONE || jurisdictionLevel == JurisdictionLevel.NATION;
+	}
 
 	@Override
 	public List<WeeklyReportOfficerSummaryDto> getSummariesPerOfficer(RegionReferenceDto regionRef, EpiWeek epiWeek) {

--- a/sormas-backend/src/main/java/de/symeda/sormas/backend/report/WeeklyReportFacadeEjb.java
+++ b/sormas-backend/src/main/java/de/symeda/sormas/backend/report/WeeklyReportFacadeEjb.java
@@ -155,8 +155,9 @@ public class WeeklyReportFacadeEjb implements WeeklyReportFacade {
 
 	@Override
 	public List<WeeklyReportRegionSummaryDto> getSummariesPerRegion(EpiWeek epiWeek) {
+		JurisdictionLevel jurisdictionLevel = userService.getCurrentUser().getJurisdictionLevel();
 
-		if (userService.getCurrentUser().getJurisdictionLevel() != JurisdictionLevel.NATION) {
+		if (!isNationOrNoneJurisdiction(jurisdictionLevel)) {
 			return new ArrayList<>();
 		}
 
@@ -204,6 +205,10 @@ public class WeeklyReportFacadeEjb implements WeeklyReportFacade {
 
 		return summaryDtos;
 	}
+
+	public boolean isNationOrNoneJurisdiction(JurisdictionLevel jurisdictionLevel) {
+       return jurisdictionLevel == JurisdictionLevel.NONE || jurisdictionLevel == JurisdictionLevel.NATION;
+    }
 
 	@Override
 	public List<WeeklyReportOfficerSummaryDto> getSummariesPerOfficer(RegionReferenceDto regionRef, EpiWeek epiWeek) {

--- a/sormas-ui/src/main/java/de/symeda/sormas/ui/reports/ReportsView.java
+++ b/sormas-ui/src/main/java/de/symeda/sormas/ui/reports/ReportsView.java
@@ -62,7 +62,7 @@ public class ReportsView extends AbstractView {
 		super(VIEW_NAME);
 
 		JurisdictionLevel jurisdictionLevel = UserRole.getJurisdictionLevel(Objects.requireNonNull(UserProvider.getCurrent()).getUserRoles());
-		if (isNationOrNoneJurisdiction(jurisdictionLevel)) {
+		if (jurisdictionLevel == JurisdictionLevel.NONE || jurisdictionLevel == JurisdictionLevel.NATION) {
 			grid = new WeeklyReportRegionsGrid();
 		} else {
 			grid = new WeeklyReportOfficersGrid();
@@ -79,10 +79,6 @@ public class ReportsView extends AbstractView {
 		gridLayout.setExpandRatio(grid, 1);
 
 		addComponent(gridLayout);
-	}
-
-	public boolean isNationOrNoneJurisdiction(JurisdictionLevel jurisdictionLevel) {
-		return jurisdictionLevel == JurisdictionLevel.NONE || jurisdictionLevel == JurisdictionLevel.NATION;
 	}
 
 	public HorizontalLayout createFilterBar() {

--- a/sormas-ui/src/main/java/de/symeda/sormas/ui/reports/ReportsView.java
+++ b/sormas-ui/src/main/java/de/symeda/sormas/ui/reports/ReportsView.java
@@ -81,9 +81,9 @@ public class ReportsView extends AbstractView {
 		addComponent(gridLayout);
 	}
 
-    public boolean isNationOrNoneJurisdiction(JurisdictionLevel jurisdictionLevel) {
-        return jurisdictionLevel == JurisdictionLevel.NONE || jurisdictionLevel == JurisdictionLevel.NATION;
-    }
+	public boolean isNationOrNoneJurisdiction(JurisdictionLevel jurisdictionLevel) {
+		return jurisdictionLevel == JurisdictionLevel.NONE || jurisdictionLevel == JurisdictionLevel.NATION;
+	}
 
 	public HorizontalLayout createFilterBar() {
 		HorizontalLayout filterLayout = new HorizontalLayout();

--- a/sormas-ui/src/main/java/de/symeda/sormas/ui/reports/ReportsView.java
+++ b/sormas-ui/src/main/java/de/symeda/sormas/ui/reports/ReportsView.java
@@ -19,6 +19,7 @@ package de.symeda.sormas.ui.reports;
 
 import java.util.Date;
 import java.util.List;
+import java.util.Objects;
 
 import com.vaadin.icons.VaadinIcons;
 import com.vaadin.navigator.ViewChangeListener.ViewChangeEvent;
@@ -60,7 +61,8 @@ public class ReportsView extends AbstractView {
 	public ReportsView() {
 		super(VIEW_NAME);
 
-		if (UserRole.getJurisdictionLevel(UserProvider.getCurrent().getUserRoles()) == JurisdictionLevel.NATION) {
+		JurisdictionLevel jurisdictionLevel = UserRole.getJurisdictionLevel(Objects.requireNonNull(UserProvider.getCurrent()).getUserRoles());
+		if (isNationOrNoneJurisdiction(jurisdictionLevel)) {
 			grid = new WeeklyReportRegionsGrid();
 		} else {
 			grid = new WeeklyReportOfficersGrid();
@@ -78,6 +80,10 @@ public class ReportsView extends AbstractView {
 
 		addComponent(gridLayout);
 	}
+
+    public boolean isNationOrNoneJurisdiction(JurisdictionLevel jurisdictionLevel) {
+        return jurisdictionLevel == JurisdictionLevel.NONE || jurisdictionLevel == JurisdictionLevel.NATION;
+    }
 
 	public HorizontalLayout createFilterBar() {
 		HorizontalLayout filterLayout = new HorizontalLayout();

